frontend/src/v5/ui/controls/formSelect/formSelect.styles.ts

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
Lets get rid of this file as it is empty

Reply:
The file is no longer empty